src/App.java

```diff
@@ -10,6 +10,13 @@
     private void start() {
         game.start();
 
+        PendingMove pendingMove = getPendingMove();
```

Comment:
It may be easier here to simply call the constructor directly. It is then obvious that `pendingMove` could never be `null`, and there will be no reason to check for it. 

Also, why include this code in `App` at all before we've figured out how to use it?